examples/kubernetes-in-docker/README.md

```diff
@@ -0,0 +1,333 @@
+[Kubernetes Conjur Demo](https://github.com/conjurdemos/kubernetes-conjur-demo)
+Github respository. These scripts can be used to deploy a few instances of
+the [pet store](https://github.com/conjurdemos/pet-store-demo/) demo
+application to demonstrate retrieval of secrets from a Conjur OSS server.
+
+Using these scripts, you can deploy the
+[pet store](https://github.com/conjurdemos/pet-store-demo/) demo application
+using the following authentication broker/clients:
+
+- [Secretless Broker](https://github.com/cyberark/secretless-broker) sidecar
+  container to manage the database connection
+- [Conjur Kubernetes Authenticator Client](https://github.com/cyberark/conjur-authn-k8s-client)
+  sidecar container to provide the Conjur access token, and Summon to inject
+  the database credentials into the application environment
+- [Conjur Kubernetes Authenticator Client](https://github.com/cyberark/conjur-authn-k8s-client)
+  init container to provide the Conjur access token, and Summon to inject
+  the database credentials into the application environment
+
+The demo script workflow can be split into three phases:
+
+- **Demo preparation**
+  - Check required environment settings
+- **Security Admin steps**
+  These steps are normally performed by a security admin, e.g.:
+  - Create a Conjur CLI deployment for scripts to use
+  - Load Conjur policies for the application
+  - Initialize Conjur's TLS Certificate Authority (CA)
+- **Demo Application Deployment**
+  - Create demo namespace
+  - Create RoleBinding to allow Conjur to access resources in the namespace
+  - Retrieve the Conjur CA certificate to add to applications trust store
+  - Build and push application containers
+  - Deploy the application instances using the Conjur authentication
+    mechanisms described above.
+
+### The Conjur Kubernetes Authenticator and Conjur Application Identity
+
+The Conjur authentication broker/clients listed above are essentially
+authentication proxies for the application. They authenticate with
+Conjur for the application using the 
+[Conjur Kubernetes Authenticator](https://docs.conjur.org/Latest/en/Content/Operations/Services/k8s_auth.htm)
+(a.k.a. `authn-k8s`) authentication plugin that must be enabled on the Conjur
+server.
+
+The `authn-k8s` plugin makes use of several forms of 
+[Conjur application identity](https://docs.conjur.org/Latest/en/Content/Integrations/Kubernetes_AppIdentity.htm?TocPath=Integrations%7COpenShift%252C%20Kubernetes%252C%20and%20GKE%7C_____2)
+to positively identify the application. In the case of Kubernetes Conjur demo
+scripts, the application identity that is used are the following
+attributes of the application instance:
+- Kubernetes Namespace name
+- Kubernetes ServiceAccount name
+- Kubernetes Deployment name
+If all three of the above resource names match what has been "white-listed"
+in Conjur policy (e.g. by a security administrator), then the application is
+permitted to access secrets as dictated by that policy.
```

Comment:
```suggestion
If all three of the above resource names match what has been specified
in Conjur policy (e.g. by a security administrator), then the application is
permitted to access secrets as dictated by that policy.
```

When I read this I wondered if any of the attributes could be a list. Based on the [docs](https://docs.conjur.org/Latest/en/Content/Integrations/Kubernetes_AppIdentity.htm?TocPath=Integrations%7COpenShift%252C%20Kubernetes%252C%20and%20GKE%7C_____2) it looks like they are definitely singular values. I thought `specified` captures this better.

Reply:
Ah, yeah, that will be less confusing.